Tasks/VSMobileCenterTest/task.json

```diff
@@ -0,0 +1,354 @@
+        },
+        {
+            "name": "app",
+            "type": "filePath",
+            "label": "Binary File Path",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Relative path from the repo root to the APK or IPA file you want to test."
+        },
+        {
+            "name": "devices",
+            "type": "string",
+            "label": "Devices",
+            "required": true,
+            "defaultValue": "",
+            "helpMarkDown": "String to identify what devices this test will run against.  Copy and paste this string when you define a new test run from Mobile Center Test beacon."
+        },
+        {
+            "name": "series",
+            "type": "string",
+            "label": "Test Series",
+            "defaultValue": "master",
+            "required": false,
+            "helpMarkDown": "The series name for organizing test runs (e.g. master, production, beta)."
+        },
+        {
+            "name": "dsymDir",
+            "type": "string",
+            "label": "DSym Directory",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Path to iOS symbol files"
+        },
+        {
+            "name": "locale",
+            "type": "pickList",
+            "label": "System Language",
+            "defaultValue": "en_US",
+            "required": true,
+            "options": {
+                "da_DK": "Danish (Denmark)",
+                "nl_NL": "Dutch (Netherlands)",
+                "en_GB": "English (United Kingdom)",
+                "en_US": "English (United States)",
+                "fr_FR": "French (France)",
+                "de_DE": "German (Germany)",
+                "ja_JP": "Japanese (Japan)",
+                "ru_RU": "Russian (Russia)",
+                "es_MX": "Spanish (Mexico)",
+                "es_ES": "Spanish (Spain)",
+                "user": "Other"
+            },
+            "helpMarkDown": "If your language isn't displayed, select 'Other' and enter its locale below, such as en_US."
+        },
+        {
+            "name": "async",
+            "type": "boolean",
+            "label": "Do not wait for test result",
+            "defaultValue": false,
+            "required": false,
+            "helpMarkDown": "Execute command asynchronously, exit when tests are uploaded, without waiting for test results."
+        },
+        {
+            "name": "userDefinedLocale",
+            "type": "string",
+            "label": "Other Locale",
+            "defaultValue": "",
+            "required": false,
+            "visibleRule": "locale = user",
+            "helpMarkDown": "Enter any two-letter ISO-639 language code along with any two-letter ISO 3166 country code in the format [language]_[country], such as en_US."
+        },
+        {
+            "name": "framework",
+            "type": "pickList",
+            "label": "Test Framework",
+            "defaultValue": "appium",
```

Comment:
Is "appium" the most popular or the first in alphabetical order? I would have picked "UI Test" since that is the type of test project created when a Xamarin project is created in VS.